src/requirements.txt

```diff
@@ -1,7 +1,7 @@
-aiohttp==3.9.5 ; python_version >= "3.11" and python_version < "4.0"
+aiohttp==3.10.11 ; python_version >= "3.11" and python_version < "4.0"
 aiosignal==1.3.1 ; python_version >= "3.11" and python_version < "4.0"
 attrs==23.2.0 ; python_version >= "3.11" and python_version < "4.0"
 frozenlist==1.4.1 ; python_version >= "3.11" and python_version < "4.0"
-idna==3.6 ; python_version >= "3.11" and python_version < "4.0"
+idna==3.7 ; python_version >= "3.11" and python_version < "4.0"
 multidict==6.0.5 ; python_version >= "3.11" and python_version < "4.0"
-yarl==1.9.4 ; python_version >= "3.11" and python_version < "4.0"
+yarl>=1.12.0,<2.0 ; python_version >= "3.11" and python_version < "4.0"
```

Comment:
Is this syntax correct? How did you determine which version to use?

Reply:
Yeah we can use this syntax
<img width="950" alt="Screenshot 2024-11-26 at 12 12 25 PM" src="https://github.com/user-attachments/assets/911ceccc-0227-4b1e-aa64-1e9cad4bbc0e">